src/orchestrator/graph.py

```
@@ -22,12 +22,19 @@
+    Configuration as ArchitectConfiguration,
+)
+from architect.graph import ArchitectGraph
+from architect.state import State as ArchitectState
 from common.graph import AgentGraph
 from orchestrator import stubs, tools
 from orchestrator.configuration import (
+    ArchitectAgentConfig,
     Configuration,
     RequirementsAgentConfig,
     SubAgentConfig,
+    ArchitectAgentConfig
```

Ruff (0.11.9)

Redefinition of unused `ArchitectAgentConfig` from line 33

Remove definition: `ArchitectAgentConfig`

(F811)